src/AutoRest.CSharp/AutoRest/Plugins/CSharpProj.cs

```diff
@@ -39,7 +39,7 @@ internal class CSharpProj : IPlugin
   </PropertyGroup>
 
   <ItemGroup>
-	  <PackageReference Include=""Microsoft.Azure.AutoRest.CSharp"" Version=""{0}"" />
+	  <PackageReference Include=""Microsoft.Azure.AutoRest.CSharp"" Version=""{0}"" PrivateAssets=""All"" />
```

Comment:
tabs are still here :)

Reply:
I don't see it. Where?

Reply:
![image](https://user-images.githubusercontent.com/1697911/107575337-ae6b2c80-6ba4-11eb-9b83-27e5ed2c8f52.png)

That arrow is a tab.